karma.conf.js

```diff
@@ -1,49 +1,18 @@
-process.env.ethTest = 'BasicTests'
-
-module.exports = function (config) {
+module.exports = function(config) {
   config.set({
     browserNoActivityTimeout: 60000,
     frameworks: ['browserify', 'detectBrowsers', 'tap'],
-    files: [
-      './tests/difficulty.js'
-    ],
+    files: ['./test-build/test/index.js'],
     preprocessors: {
-      'tests/*.js': ['browserify', 'env']
+      './test-build/**/*.js': ['browserify']
     },
     singleRun: true,
-    plugins: [
-      'karma-browserify',
-      'karma-chrome-launcher',
-      'karma-env-preprocessor',
-      'karma-tap',
-      'karma-firefox-launcher',
-      'karma-detect-browsers'
-    ],
-    browserify: {
-      'transform': [
-        [
-          'babelify',
-          {
-            'presets': [
-              'env'
-            ]
-          }
-        ]
-      ]
-    },
     detectBrowsers: {
       enabled: true,
       usePhantomJS: false,
-      postDetection: function (availableBrowser) {
-        if (process.env.TRAVIS) {
-          return ['Firefox']
-        }
-
-        var browsers = ['Chrome', 'Firefox']
-        return browsers.filter(function (browser) {
-          return availableBrowser.indexOf(browser) !== -1
-        })
-      }
+      postDetection: function(availableBrowsers) {
+        return ['Firefox']
+      },
     }
   })
 }
```

Comment:
This may be my fault, I copied part of the config from the `-tx` migration.

Reply:
I think they were disabled because travis was failing because the tests' output exceeded the maximum log that they are willing to store. I tried different things and finally made it work by changing the Karma reporters setting.